Ladybird/AppKit/UI/Tab.mm

```diff
@@ -91,8 +91,8 @@ - (instancetype)init
         [self.search_panel setHidden:YES];
 
         auto* scroll_view = [[NSScrollView alloc] init];
```

Comment:
I suspect we could switch from NSScrollView + NSClipView to just NSView as now scroll handling is completely moved into WebContent, but I am going to leave it as an opportunity for future refactoring :)